Mathlib/Logic/Relation.lean

```diff
@@ -401,6 +403,17 @@ theorem head'_iff : TransGen r a c ↔ ∃ b, r a b ∧ ReflTransGen r b c := by
   rcases IH with ⟨d, had, hdb⟩
   exact ⟨_, had, hdb.tail hbc⟩
 
+theorem _root_.Subrelation.transGen {s : α → α → Prop} [IsTrans α s] (h : Subrelation r s) :
```

Comment:
Following the rest of this file, it looks like this shouldn't be stated with `Subrelation` and `IsTrans`, rather `Transitive` and an explicit forall, see `reflGen_minimal`, for example. But we might as well have both.